### PR TITLE
kubernetes: Add Pending volume claims

### PR DESCRIPTION
**Pull request by petervo**

### CODE REVIEW

**Comment by stefwalter** [comment]
This needs a rebase now that #4422 was merged.

File: pkg/kubernetes/scripts/volumes.js [hunk 1]
```diff
@@ -872,6 +907,34 @@
                 return defer.promise;
             }
 
+            function updatePVC() {
+                /*
+                 * HACK: Until the API gets a way to manually
```
Comment (stefwalter) [inline pkg/kubernetes/scripts/volumes.js]: Is there a bug filed upstream for this?

Comment (stefwalter) [inline pkg/kubernetes/scripts/volumes.js]: I can add the bug link while merging if you want. But just need it to be filed.

Comment (petervo) [inline pkg/kubernetes/scripts/volumes.js]: There's this bug, https://github.com/kubernetes/kubernetes/issues/21498. Basically this code being rewritten right now in kubernetes, Once the refactor lands I'll reevaluate and see if further bugs need to be opened.

**Pull request closed without merging**